src/ProductCard/StoreLocationButton.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from 'react'
+import ChevronRight from '../Icons/ChevronRight'
+import MapMarker from '../Icons/MapMarker'
+import clsx from 'clsx'
+
+import styles from './productCard.module.scss'
+
+interface Props {
+  storeLocation?: string
+  onClick?: Function
+}
+
+const StoreLocationButton: FunctionComponent<Props> = ({ storeLocation, onClick }) => {
+  const handleClick = () => {
```

Comment:
this function shouldn't exist. Just pass `onClick` to the onClick on the button comp. 

```
<button onClick={onClick}>
```

Reply:
if the the prop onClick is undefined, then the event listener won't be passed the Element